automatic/1password/tools/chocolateyInstall.ps1

```diff
@@ -3,10 +3,8 @@
 $packageArgs = @{
   packageName    = $env:ChocolateyPackageName
   fileType       = 'exe'
-  url            = 'https://c.1password.com/dist/1P/win6/1PasswordSetup-7.9.832.exe'
+  url            = 'https://downloads.1password.com/win/1PasswordSetup-latest.exe'
   softwareName   = '1Password*'
-  checksum       = '578022880cbb9f9dc1e0d61e77077ad105e51727ad09f2565fe512ba06107611'
```

Comment:
If you specify the specific URL, this can be added back in with the appropriate checksum for the version, in this case:
73F3193FC8AC34FDDB4CB5EF2F4A6986CE29DFEB148DA1ABE951E432FDD388DE